babel/.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["@babel/preset-env"]
+}
```

Comment:
Should this file be replaced with "babel.config.js"? Untested content below...
```js
module.exports = () => {
  const presets = [
      [
         "@babel/preset-env", 
         { 
             useBuiltIns: false
         }
      ]
   ];

   return { presets };
};
```

Reply:
Yes I think so.